saleor/graphql/attribute/tests/queries/test_attribute_filter.py

```diff
@@ -554,7 +554,7 @@ def test_filter_attributes_in_category_not_published_by_staff_without_manage_pro
     )["data"]["attributes"]["edges"]
 
     # then
-    assert len(attributes) == attribute_count
+    assert len(attributes) == attribute_count - 1
```

Comment:
We should change the way `attribute_count` is calculated. @fowczarek Do you have any idea what would be the best way to do it?

Reply:
Maybe we should introduce some helper for calculating `attribute_count` based on the products list? 
@maarcingebala  IMO we could do it in separate PR.